frontend/src/metabase/nav/components/search/SearchBar/SearchBar.styled.tsx

```diff
@@ -38,7 +38,6 @@ export const SearchInputContainer = styled.div<{
     }
     return css`
       background-color: var(--mb-color-bg-white);
-
```

Comment:
personally, I'd remove this change to ensure the diff is clean, but 🤷

Reply:
Ok, this was because stylelint which is run by lintstaged.

We can verify with `yarn stylelint --customSyntax postcss-styled-syntax --fix frontend/src/metabase/nav/components/search/SearchBar/SearchBar.styled.tsx`, it will remove this line, so this is already correct.